python/ray/_private/metrics_agent.py

```diff
@@ -371,6 +379,18 @@ def to_metric(
             )
             return metric
 
+        elif isinstance(agg_data, SumAggregationData):
+            metric = metrics_map.get(metric_name)
+            if not metric:
+                metric = CounterMetricFamily(
```

Comment:
So apparently GaugeMetricFamily (gauge), CounterMetricFamily (Count) and UnknownMetricFamily (Sum) from prometheus is modifying the metrics name...
For example `internal_num_processes_skipped_job_mismatch` which initially in `GaugeMetricFamily` since it's a type Sum becomes `internal_num_processes_skipped_job_mismatch_workers_total` when it's type is changed to `Count` corresponds to CounterMetricFamily

is it okay to change the metrics name resulted from this? Otherwise `ray.utils.metrics.Counter` and any `Sum` stats like `internal_num_processes_skipped_job_mismatch` will be treated as untyped